pcsx2/IopBios.cpp

```diff
@@ -611,15 +611,30 @@ namespace R3000A
 			return 0;
 		}
 
+		int remove_HLE()
+		{
+			const std::string full_path = clean_path(Ra0);
+
+			if (is_host(full_path))
+			{
+				const std::string path = full_path.substr(full_path.find(':') + 1);
+				const ghc::filesystem::path file_path{host_path(path)};
+				const bool suceeds = ghc::filesystem::remove(file_path);
+				v0 = suceeds ? 0 : -IOP_EIO;
+				pc = ra;
+			}
+			return 0;
+		}
+
 		int mkdir_HLE()
 		{
 			const std::string full_path = clean_path(Ra0);
 
 			if (is_host(full_path))
 			{
 				const std::string path = full_path.substr(full_path.find(':') + 1);
-				ghc::filesystem::path folder_path{host_path(path)};
-				bool suceeds = ghc::filesystem::create_directory(folder_path);
+				const ghc::filesystem::path folder_path{host_path(path)};
+				const bool suceeds = ghc::filesystem::create_directory(folder_path);
```

Comment:
typo, succeeds would be correct, but succeeded might be a better way to word it?

Reply:
Applied